### PR TITLE
Added the resource link

### PR DESCRIPTION
**Pull request by HarishmaA**
Add yourself and your favorite resources to the project #1
Fork the project
Make a copy of the template in the README.md and paste it to the bottom
Use the template to add in more resources

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,3 +62,13 @@ A Repository to collect and showcases developers' favorite resources to learn a
     - The interactivity of the project is what helped it work for me. It is a lot easier to understand something when you're able to actually use it and see the effects.
     - For complete beginners it might be difficult to translate the lessons from this site directly to a real project, especially one where you are collaborating with other people.
 
+[Harishma.A](https://github.com/HarishmaA) 
+> Software Engineer working at Full Creative
+
+#### UDACITY
```
Comment (bhavaniravi) [inline README.md:68]: We are categorizing resources based on tech-stack and not platforms. I would like you to add specific courses and tag what you like about them. For eg., people who like Java can pick it up from there

**Review by bhavaniravi — CHANGES_REQUESTED** [review]
Hi Harishma, Thank you for your contribution. Here are a few things we need to change

**Comment by bhavaniravi** [comment]
@HarishmaA Please let me know if you can update the PR

File: README.md (hunk 1 above)
Comment (HarishmaA) [reply README.md]: Sure.. Sorry for late reply..

Comment (HarishmaA) [reply README.md:68]: Sure.. I will add specific courses and sorry for the late reply.

**Comment by HarishmaA** [comment]
> @HarishmaA Please let me know if you can update the PR

Will update it soon

**Comment by HarishmaA** [comment]
I have created PR with specific course for ReactJS

**Comment by HarishmaA** [comment]
Thank you

**Pull request merged**